mapper/pojo-base/src/main/java/org/hibernate/search/mapper/pojo/massindexing/spi/PojoMassIndexingMappingContext.java

```diff
@@ -33,6 +33,7 @@ public interface PojoMassIndexingMappingContext extends BackendMappingContext {
 	 * @param context A context with information about the mass indexing that is about to start.
 	 * @return An agent.
 	 */
+	// TODO: Yoann, feels like we are talking about outbox-polling here that should work only with ORM? drop the example?
```

Comment:
IMO we can accept talking about concepts that only make sense for ORM if it helps with implementation of those SPIs.

Regardless, you could just replace "(e.g. automatic indexing)" with "(e.g. indexing plans)", since explicit indexing through indexing plan would also be suspended.